resources/views/faults/list.blade.php

```diff
@@ -0,0 +1,98 @@
+<div id="faults-table"></div>
+<script src="js/moment.min.js"></script>
+<script type="application/javascript">
+    $(document).ready(function () {
+        function status_translate(cell) {
+            if (typeof cell !== "string") {
+                cell = cell.getValue();
+            }
+            var translations = {
+                UNSEEN: "@lang('faults.unseen')",
+                SEEN: "@lang('faults.seen')",
+                DONE: "@lang('faults.done')",
+                WONT_FIX: "@lang('faults.wont_fix')"
+            };
+            var colors = {
+                UNSEEN: "text-secondary",
+                SEEN: "text-info",
+                DONE: "text-success",
+                WONT_FIX: "text-danger"
+            };
+            return '<div class="font-italic ' + colors[cell] + ' text-right">' + translations[cell] + '</div>';
+        }
+
+        var button = function (cell, formatterParams) {
+            if (formatterParams['status'] === "DONE") {
+                var style = "btn-success";
+                var text = "@lang('faults.done')";
+            } else if (formatterParams['status'] === "WONT_FIX") {
+                var style = "btn-danger";
+                var text = "@lang('faults.wont_fix')";
+            } else if (formatterParams['status'] === "UNSEEN") {
+                var style = "btn-danger";
+                var text = "@lang('faults.reopen')";
+            }
+            return $("<button type=\"button\" class=\"btn btn-sm " + style + " float-left\">" + text + "</button>").click(function () {
+                var data = cell.getRow().getData();
```

Comment:
Prefer `const`, `let` over `var` variable declarations.
I mentioned this only one despite you use `var` everywhere in this patch.

Reply:
I've changed constants to `const`.
If you think I should, I can change `var`s to `let`s, but keep in mind that `let` isn't used anywhere  here, every JavaScript program uses `var` (e.g. [this](https://github.com/luksan47/mars/blob/uran-2.x/resources/views/admin/internet/internet_access/list.blade.php), [this](https://github.com/luksan47/mars/blob/uran-2.x/resources/views/admin/internet/mac_addresses/list.blade.php) or [this](https://github.com/luksan47/mars/blob/uran-2.x/resources/views/print/history.blade.php)).

Reply:
I will let you decide if you can use `let` for declaring the given variable or not.
I'm just saying that `let` should be preferred to `var` in any circumstances.
To learn more about this: https://stackoverflow.com/a/11444416

Btw yes, there are other cases where do author did not pay attention to these guidelines, this happens.